ansible_collections/arista/avd/roles/eos_validate_state/tasks/routing_table.yml

```diff
@@ -17,7 +17,7 @@
   assert:
     that:
       - vtep_reachability_test['stdout'][0] | regex_search(vtep_reachability_test.vtep_ip)
-    fail_msg: "VTEP {{ vtep_reachability_test.vtep_ip }} is not in the routing table"
+    fail_msg: "VTEP {{ vtep_reachability_test.vtep_ip | replace('\"','') }} is not in the routing table"
```

Comment:
Any reason to use replace('\"','')?

Because the template generate_state_report.j2 (which was passed into the yaml_templates_to_facts) is having these double quotes in the templates itself.